Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -141,13 +141,16 @@ def adjoint(self,x, out=None):
         raise NotImplementedError
     
     @staticmethod
-    def PowerMethod(operator, max_iteration=10, initial=None, tolerance = 1e-5,  return_all=False):
+    def PowerMethod(operator, max_iteration=10, initial=None, tolerance=1e-5,  return_all=False, symmetric=False):
 
         r"""Power method or Power iteration algorithm 
         
         The Power method computes the largest (dominant) eigenvalue of a square matrix in magnitude, e.g.,
         absolute value in the real case and module in the complex case.        
-        For the non-square case, the power method is applied for the matrix :math: A^{T}*A.
+        Since we cannot estimate if the operator is diagonisable, the power method is applied on the diagonizable operator
+        :math:`A^{T}*A`.
+        
+        If the operator :math:`A` is diagonizable, the user can pass the flag `symmetric=True`.

```

Comment:
```suggestion
        Since we cannot estimate if the operator is diagonisable, the power method is applied on the diagonizable operator
        :math:`A^{T}*A`.
        
        If the operator :math:`A` is diagonisable, the user can pass the flag `symmetric=True`.

```